crud/program/src/instruction.rs

```diff
@@ -0,0 +1,135 @@
+//! Program instructions
+
+use crate::{state::Data, *};
+use borsh::{BorshDeserialize, BorshSerialize};
+use solana_program::{
+    instruction::{AccountMeta, Instruction},
+    pubkey::Pubkey,
+    sysvar,
+};
+
+/// Instructions supported by the Feature Proposal program
+#[derive(Clone, Debug, BorshSerialize, BorshDeserialize, PartialEq)]
+pub enum CrudInstruction {
+    /// Create a new document
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    /// 0. `[writeable, signer]` Data account, must be uninitialized
+    /// 1. `[]` Document owner
+    /// 2. `[]` Rent sysvar, to check for rent exemption
+    Create {
+        /// Data to be filled into the account
+        data: Data,
+    },
+
```

Comment:
I think we'll want a `SetAuthority` instruction as well

Reply:
Done in f63408d

Reply:
lgtm